_includes/snippets/14/gcc_example.snip

```diff
@@ -1,73 +1,153 @@
+{{ site.host_prompt }} module load matlab/full/r2017a/9.2
+```
+{: .bash}
+```
+~/.matlab is a symbolic link pointing to /home/cceagil/Scratch/.matlab
 
-The following have been reloaded with a version change:
-  1) gcc/5.4.0 => gcc/4.8.5
+Matlab setup complete type matlab to start Matlab.
+```
+{: .output}
 
-gcc (GCC) 4.8.5
-Copyright (C) 2015 Free Software Foundation, Inc.
-This is free software; see the source for copying conditions.  There is NO
-warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+```
+{{ site.host_prompt }} matlab -nodisplay -nosplash -nodesktop
+```
+
+{: .bash}
+```
+                                               < M A T L A B (R) >
+                                     Copyright 1984-2017 The MathWorks, Inc.
+                                      R2017a (9.2.0.556344) 64-bit (glnxa64)
+                                                  March 27, 2017
+
+ 
+To get started, type one of these: helpwin, helpdesk, or demo.
+For product information, visit www.mathworks.com.
+ 
+>> quit
 ```
 {: .output}
 
-We now have successfully switched from GCC 5.4.0 to GCC 4.8.5. It is also important to note that
-there was no compatible OpenMPI module available for GCC 4.8.5. Because of this, the `module`
-program has "inactivated" the module. All this means for us is that if we re-load GCC 5.4.0,
-`module` will remember OpenMPI used to be loaded and load that module as well.
+Note that you cannot load two different versions of the same software at once.
+Currently, we have loaded `matlab/full/r2017a/9.2`. Let's try also loading
+`matlab/full/r2015b/8.6`:
 
 ```
-{{ site.host_prompt }} module load gcc/5.4.0
+{{ site.host_prompt }} module load matlab/full/r2015b/8.6
 ```
 {: .bash}
 ```
-Activating Modules:
-  1) openmpi/2.1.1
+matlab/full/r2015b/8.6(108):ERROR:150: Module 'matlab/full/r2015b/8.6' conflicts with the currently loaded module(s) 'matlab/full/r2017a/9.2'
+matlab/full/r2015b/8.6(108):ERROR:102: Tcl command execution failed: conflict matlab
+```
+{: .output}
 
-The following have been reloaded with a version change:
-  1) gcc/4.8.5 => gcc/5.4.0
+As we can see, we get an error message about conflicts.
+If we do indeed wish to load version r2015b/8.6, we can say
+```
+{{ site.host_prompt }} module unload matlab/full/r2017a/9.2
+{{ site.host_prompt }} module load matlab/full/r2015b/8.6
+```
+or, in one step:
+```
+{{ site.host_prompt }} module swap matlab matlab/full/r2015b/8.6
+```
+```
+{{ site.host_prompt }} module list
+```
+{: .bash}
+```
+Currently Loaded Modulefiles:
+  1) gcc-libs/4.9.2                 9) gerun                         17) userscripts/1.4.0
+  ...                              ...                               ...
+  6) apr-util/1.5.4                14) emacs/24.5                    22) xorg-utils/X11R7.7
+  7) subversion/1.8.13             15) tmux/2.2                      23) matlab/full/r2015b/8.6
+  8) screen/4.2.1                  16) mrxvt/0.5.4
 ```
 {: .output}
+
+
```

Comment:
This file is twice in this PR. Why are you editing also the one under `_include`? This file should not be touched. Get this file from `gh-pages` so the edits are not included here.

Reply:
Sorry, now fixed.